source/_integrations/zha.markdown

```diff
@@ -51,21 +51,21 @@ ZHA uses an open-source Python library implementing a hardware-independent Zigbe
 
 There is currently support for the following device types within Home Assistant:
 
-- Alarm Control Panel
-- Binary Sensor
-- Button
-- Climate (beta)
-- Cover
-- Fan
-- Light
-- Lock
-- Number (i.e. analog output)
-- Select
-- Sensor
-- Siren
-- Switch
-
-There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
+- [Alarm Control Panel](../alarm_control_panel/)
+- [Binary Sensor](../binary_sensor/)
+- [Button](../button/)
+- [Climate](../climate/) (beta)
+- [Cover](../cover/)
+- [Fan](../fan/)
+- [Light](../light/)
+- [Lock](../lock/)
+- [Number](../number/) (i.e. analog output)
+- [Select](../select/)
+- [Sensor](../sensor/)
+- [Siren](../siren/)
+- [Switch](../switch/)
+
```

Comment:
Do not use these links as such. It should be links relative to the root.

For example, not `../binary_sensor`, but `/integrations/binary_sensor`.

Please correct this.

Reply:
@frenck @c0ffeeca7  already merged so hoping this new PR can fix it? -> https://github.com/home-assistant/home-assistant.io/pull/27884